src/runtime/runtime_stm32.go

```diff
@@ -4,12 +4,55 @@ package runtime
 
 import (
 	"device/arm"
+	"device/stm32"
+	"machine"
 )
 
 type timeUnit int64
 
 const tickMicros = 1 // TODO
 
+const (
+	// Flash Access Control Register flag values.
+	FLASH_ACR_LATENCY_0 = 0x00000004
```

Comment:
This value looks suspicious. Shouldn't this be 0x00000001?

Reply:
Yes, you are correct.